### PR TITLE
Use object not boolean params

### PR DESCRIPTION
**Pull request by jeremyhansonfinger**
@nwtn My revised index.js passes my revised test.js. Will have to run the rest of the `npm test` suite later. If you have a chance to work on it tonight, awesome. Do whatever you want!

### CODE REVIEW

File: test.js [hunk 1]
```diff
@@ -75,7 +75,7 @@ test('Case', function (t) {
     t.end();
 });
 
-test('Apostrophes', function (t) {
+test('Apostrophes', function (t, options) {
```
Comment (nwtn) [inline test.js]: you don’t need `options` here, it’s never used

File: test.js [hunk 2]
```diff
@@ -497,15 +497,15 @@ test('Apostrophes', function (t) {
...
+        ], {allowApostrophes: true}),
         'don\'t blocklevel',
         'should normalize smart apostrophes (multiple nodes) if true'
     );
 
     t.end();
 });
 
-test('Dashes', function (t) {
+test('Dashes', function (t, options) {
```
Comment (nwtn) [inline test.js]: same here

**Comment by nwtn** [comment]
Looks dope. A couple comments, then make sure `npm test` (which runs more than just `node test` does) passes and I think you’re good!

**Comment by jeremyhansonfinger** [comment]
@nwtn awww yiss

image

**Comment by jeremyhansonfinger** [comment]
image

**Pull request merged**